client/src/styles/Admin.css

```diff
@@ -0,0 +1,107 @@
+.admin-section-wrapper {
+  background-image: linear-gradient(rgba(41, 42, 99, 1), rgba(255, 212, 76, 0)),
+    url("../assets/images/backgroundRegister.png");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: 100svh;
+  padding-bottom: 6rem;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.admin-section-information-desktop {
+  display: none;
+}
+
+.admin-section {
+  width: 98%;
+  background-color: var(--fourth-color);
+  margin-top: 3.2rem;
+  border: 1px solid var(--fifth-color);
+  border-radius: 0.8125rem;
+}
+
+.admin-main-title {
+  color: var(--third-color);
+  font-size: 2.4rem;
+  text-align: center;
+  margin: 1.5rem 0;
+  letter-spacing: 0.2rem;
+}
+
+.admin-stats-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  color: var(--secondary-color);
+  margin-top: 1rem;
+}
+
+.admin-stats-section > article {
+  background-color: var(--primary-color);
+  width: 78%;
+  padding: 2rem;
+  text-align: center;
+  font-size: 1.5625rem;
+}
+
+.admin-stats-section > article > p:first-of-type {
+  font-weight: 600;
+  font-size: 2.2rem;
+  padding-bottom: 0.3rem;
+}
+
+.admin-stats-section > article > section {
+  font-size: 1.3rem;
+  font-style: italic;
+  padding-top: 1.2rem;
+  display: flex;
+  justify-content: center;
+  gap: 0.3rem;
+}
+
+.admin-stats-section > article > section > p:nth-of-type(2) {
+  font-weight: 600;
+}
+
+.admin-links-section {
+  margin-top: 3rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.admin-links-link {
+  width: 25rem;
+  height: 3rem;
+  border: none;
+  border-radius: 0.65rem;
+  background-color: var(--secondary-color);
+  color: var(--primary-color);
+  font-size: 1.3rem;
+  font-weight: bold;
```

Comment:
In digits please like the one you have done before: font-weight: 600

Reply:
done